api/dbadapters/snowflake.ts

```diff
@@ -185,21 +188,50 @@ where LOWER(table_schema) != 'information_schema'
     }));
   }
 
+  public async search(searchText: string): Promise<dataform.ITableMetadata[]> {
+    const results = await this.execute(
+      `select tables.table_catalog as table_catalog, tables.table_schema as table_schema, tables.table_name as table_name
+       from information_schema.tables as tables
+       left join information_schema.columns as columns on tables.table_catalog = columns.table_catalog and tables.table_schema = columns.table_schema
+         and tables.table_name = columns.table_name
+       where LOWER(tables.table_catalog) like :1 or LOWER(tables.table_schema) like :1 or LOWER(tables.table_name) like :1 or tables.comment like :1
```

Comment:
https://docs.snowflake.com/en/sql-reference/functions/ilike.html

I think most DBs have this